Commands/Commands/queue.js

```diff
@@ -426,10 +426,9 @@ commands.registerVote = {
                   }
                 }).catch(e => {
                   if (e.statusCode === 404) {
-                    logger.log(bot, {
-                      cause: 'feed_vote',
-                      message: (e.message !== undefined) ? e.message : JSON.stringify(e)
-                    }, e)
+                    bot.Channels.get(doc.channel).sendMessage("The suggestion is no longer available to be voted on").then(c => {
```

Comment:
... And I was laughing at you for messing up when copy pasting code, oh how karma is cruel.